index.d.ts

```diff
@@ -14,4 +14,5 @@ export function getBinding(service: 'REDIS', client: 'redis'): RedisClientOption
 export function getBinding(service: 'REDIS', client: 'ioredis'): RedisOptions;
 export function getBinding(service: 'MONGODB', client: 'mongodb'): MongoClientOptions;
 export function getBinding(service: 'AMQP', client: 'rhea'): rheaConnectionOptions;
-export function getBinding(service: 'MYSQL', client: 'mysql2'): mysql2ConnectionOptions;
\ No newline at end of file
+export function getBinding(service: 'MYSQL', client: 'mysql2'): mysql2ConnectionOptions;
+export function getBinding(service: 'MYSQL', client: 'mysql'): mysql2ConnectionOptions;
```

Comment:
These two shouold have different names right. My guess is you just missed removing the 2 in the second one.

Reply:
I couldn't think of any other alternative,  once mysql client does not have types. The only difference I see from [mysql client](https://github.com/sidorares/node-mysql2/blob/master/typings/mysql/lib/Connection.d.ts) with the [mysql2 client](https://github.com/sidorares/node-mysql2/blob/master/typings/mysql/lib/Connection.d.ts) is the option localInfile, rest of mysql options are a subset of mysql2 options

Reply:
I'm not comfortable using types for one client for the other. I think at this point we should just not include. We can see if anybody complains or has a better suggestion.